docs/unified-test-documentation/dasharo-performance/407-cyclic-suspend-and-resume.md

```diff
@@ -0,0 +1,83 @@
+# Dasharo Performance: Cyclic platform suspend and resume
```

Comment:
Cyclic suspend resume test is made for reliability not performance reasons. It fits into the compatibility category.

If we want suspend/resume the performance test, I would focus on "how much time does it take from issuing a waking vector to see OS login prompt or screensaver back again"

tl;dr

Please move the file content to the 31M file in compatibility as a new test case number

Reply:
@miczyg1 OK, I just moved this test case and remove references to it.

BTW, in my opinion, we should think about the new division of the test groups. At this point, I wanted to assume that the group `performance` contains not only performative tests but also reliability tests. Maybe I should prepare the proposal for the new division (ofc with a description that informs how we allocate a given test to a given group).

Reply:
One can do performance tests only when everything works reliably. So first comes reliability, then performance, not the other way around. Performance should check what it really means: speed, time, efficiency, etc.